From 3593dbf704952575294b44dbd97d3026ed7429f2 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sat, 19 Jan 2013 16:35:19 +0100
Subject: [PATCH] Remove unneeded and confusing usage of interest-noawait
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The package is already making use of noawaiting triggers by way of
«dpkg-trigger --no-await», so there's no need to complicate the upgrade
path with an unneeded Pre-Depends. In addition even if currently dpkg
will just use the last interest directive, so the interest-noawait is
w/o effect, using both for the same trigger event is just confusing
and unneeded.
---
 debian/ca-certificates.triggers | 2 --
 debian/control                  | 1 -
 2 files changed, 3 deletions(-)

diff --git a/debian/ca-certificates.triggers b/debian/ca-certificates.triggers
index 14dec6e..845da6e 100644
--- a/debian/ca-certificates.triggers
+++ b/debian/ca-certificates.triggers
@@ -1,4 +1,2 @@
-interest-noawait update-ca-certificates
-interest-noawait update-ca-certificates-fresh
 interest update-ca-certificates
 interest update-ca-certificates-fresh
diff --git a/debian/control b/debian/control
index 8f84573..29de08f 100644
--- a/debian/control
+++ b/debian/control
@@ -13,7 +13,6 @@ Vcs-Browser: http://git.debian.org/?p=collab-maint/ca-certificates.git
 
 Package: ca-certificates
 Architecture: all
-Pre-Depends: dpkg (>= 1.16.1)
 Depends: openssl (>= 1.0.0), ${misc:Depends}
 Enhances: openssl
 Multi-Arch: foreign
-- 
1.8.1.1

